src/utils/EventUtils.ts

```diff
@@ -0,0 +1,227 @@
+import { Result } from "@ethersproject/abi";
+import { delay } from "@uma/financial-templates-lib";
+import { SortableEvent } from "../interfaces";
+import { Contract, Event, EventFilter } from "ethers";
+
+const maxRetries = 3;
+const retrySleepTime = 10;
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function spreadEvent(args: Result = {} as Result): any {
```

Comment:
Set a default argument for `args: Result = {} as Result`

Reply:
For potential follow-up, we can probably refine the return type annotation. I believe this function should return a Record<string, any>. I guess that's an improvement on just returning any.

Reply:
I personally think we should leave these refactorings to after the relayer migration, but this is a great point to further move toward using a fully typed relayer

Reply:
Yea lets refactor post migration so its easier to review!